### PR TITLE
Add basic documentation for routers

### PR DESCRIPTION
**Pull request by aliciaxw**
Would appreciate any suggestions (content, amount of documentation) on the docs for simple routers before I start working on the messier ones!

#85 #86

### CODE REVIEW

File: src/routers/AlertsRouter.js [hunk 1]
```diff
@@ -3,6 +3,10 @@ import { AppDevRouter } from 'appdev';
 import type Request from 'express';
 import AlertsUtils from '../utils/AlertsUtils';
 
+/**
+ * Express Router object for retrieving TCAT alert data
```
Comment (young-k) [inline src/routers/AlertsRouter.js]:
```suggestion
 * Router object for retrieving TCAT alert data
```
Can you explain more what a "TCAT alert" is? I'm personally unsure of what it is, so I'm sure other people have the same question!

File: src/routers/AllStopsRouter.js [hunk 2]
```diff
@@ -3,6 +3,10 @@ import { AppDevRouter } from 'appdev';
 import type Request from 'express';
 import AllStopUtils from '../utils/AllStopUtils';
 
+/**
+ * Express Router object for retrieving data of all TCAT bus stops
```
Comment (young-k) [inline src/routers/AllStopsRouter.js]:
```suggestion
 * Router object for retrieving data of all TCAT bus stops
```

File: src/routers/DelayRouter.js [hunk 3]
```diff
@@ -3,6 +3,10 @@ import { AppDevRouter } from 'appdev';
 import type Request from 'express';
 import RealtimeFeedUtils from '../utils/RealtimeFeedUtils';
 
+/**
+ * Express Router object for retrieving bus delay data
```
Comment (young-k) [inline src/routers/DelayRouter.js]:
```suggestion
 * Router object for retrieving bus delay data
```

**Comment by aliciaxw** [comment]
fixed changes requested

**Pull request merged**